test/iat.tests.js

```diff
@@ -12,11 +12,4 @@ describe('iat', function () {
     expect(result.exp).to.be.closeTo(iat + expiresIn, 0.2);
   });
 
-
-  it('should throw if iat is not a number', function () {
```

Comment:
Why do you remove it? is it because with this PR it is tested in the schema test?